libvast/test/system/catalog.cpp

```diff
@@ -252,11 +277,14 @@ struct fixture : public fixtures::deterministic_actor_system_and_events {
     q += " && :timestamp <= 1970-01-01+";
     q += hhmmss_to;
     q += ".0";
-    return lookup(meta_idx, q);
+    return lookup(catalog_act, q);
   }
 
   // Our unit-under-test.
-  catalog_actor meta_idx;
+  catalog_actor catalog_act;
+
+  // Index for registering types.
+  index_actor index;
```

Comment:
Why does the catalog unit test need the index?

Reply:
From what I can tell, the `catalog taxonomies` test contains custom `vast::type`s that need to have `active partitions` registered. Now with the `type_registry` gone, I need to do it in the `index`.

Reply:
That makes sense. Thanks for explaining.